code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -302,9 +302,9 @@
 
 /obj/item/gun/projectile/shotgun/automatic
 
-/obj/item/gun/projectile/shotgun/automatic/detailed_examine()
-	return "This is a ballistic weapon. After firing, it will automatically cycle the next shell. To reload, load more shotgun \
-			shells into the gun."
+/obj/item/gun/projectile/shotgun/automatic/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>Automatically releases spent shotgun shells.</span>"
```

Comment:
This proc will call the parent and then will display some conflicting information about automatic vs manual shell ejection. One solution is to make a `shotgun_info()` or similarly named proc that returns the examine the text you want, and then you can override it for the automatic variant.
```dm
/obj/item/gun/projectile/shotgun/examine(mob/user)
	. = ..()
	. += shotgun_info()
```